test/fake-repos/fake-proto/excluded/excluded.proto

```diff
@@ -0,0 +1,8 @@
+// This is a fake proto file for testing.
```

Comment:
How does the test know whether this proto was excluded or not? Because it contains no content, would the test have the same result if this was included?